src/bundlers/dart/dartBundler.ts

```diff
@@ -197,7 +199,13 @@ export class DartBundler implements BundlerInterface {
                 }
                 // copy file to tmp folder
                 const fileDestinationPath = path.join(tempFolderPath, filePath.path);
-                return fs.promises.copyFile(filePath.path, fileDestinationPath);
+                return fs.promises.copyFile(filePath.path, fileDestinationPath).catch((error) => {
```

Comment:
i kind of don't like this solution because if we have multiple files with this problem we will have the same error a lot of times in the console